### PR TITLE
Conways Game of life

### PR DESCRIPTION
**Pull request by ankitk50**
Streamlit browser app interface for Connway's game of life from the mesa example. Run the example using - streamlit run app.py. The animated grid is modelled using pandas and Altair.

### CODE REVIEW

File: examples/conways_game_of_life/app.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+from conways_game_of_life.model import ConwaysGameOfLife
+
+import pandas as pd
+
+
+import random
+
+model = st.title("Boltzman Wealth Model")
+num_ticks = st.slider(
+    "Select number of Simulation Runs", min_value=1, max_value=100, value=50
```
Comment (rht) [inline examples/conways_game_of_life/app.py:23]: Should be "Select number of simulation steps".

Comment (ankitk50) [reply examples/conways_game_of_life/app.py]: this has been updated on the latest commit

**Comment by rht** [comment]
The merge conflict happens because in the Boltzmann wealth model PR, you accidentally included GoL app.py. Needs to rebase this PR.

**Comment by ankitk50** [comment]
> The merge conflict happens because in the Boltzmann wealth model PR, you accidentally included GoL app.py. Needs to rebase this PR.

Apologies, this is now fixed.

**Comment by rht** [comment]
You still have merge commits. This means that you haven't set up the rebase `git config pull.rebase true`.

**Comment by rht** [comment]
It's fine for now since the commits are going to be squashed anyway.

**Pull request merged**